dispatcher/backend/.flake8

```diff
@@ -0,0 +1,5 @@
+# .flake8
+
+[flake8]
+max-line-length = 88
+extend-ignore = E203
```

Comment:
We can add the extend-ignore to the flake8 command (` --extend-ignore=E203`) and thus not need this file at all

Reply:
Hum, I have a similar setup in `~/.flake8` but that's because those are what we use everywhere. Would that work for you?
I am not against having the file in the repo but then it hides this behavior from within the files that call the flake8 binary… Can be solved with a comment maybe

Reply:
Yes, good point, I will remove these files from the repo